packages/account/src/Components/financial-details/__tests__/financial-details.spec.tsx

```diff
@@ -14,48 +15,8 @@ const modal_root_el = document.createElement('div');
 modal_root_el.setAttribute('id', 'modal_root');
 document.body.appendChild(modal_root_el);
 
-const fields_enums: TFinancialInformationAndTradingExperience = {
-    account_turnover_enum: [
-        { value: 'account turnover 1', text: 'account turnover 1' },
-        { value: 'account turnover 2', text: 'account turnover 2' },
-    ],
-    education_level_enum: [
-        { value: 'education level 1', text: 'education level 1' },
-        { value: 'education level 2', text: 'education level 2' },
-    ],
-    employment_industry_enum: [
-        { value: 'employment industry 1', text: 'employment industry 1' },
-        { value: 'employment industry 2', text: 'employment industry 2' },
-    ],
-    estimated_worth_enum: [
-        { value: 'estimated worth 1', text: 'estimated worth 1' },
-        { value: 'estimated worth 2', text: 'estimated worth 2' },
-    ],
-    income_source_enum: [
-        { value: 'income source 1', text: 'income source 1' },
-        { value: 'income source 2', text: 'income source 2' },
-    ],
-    net_income_enum: [
-        { value: 'net income 1', text: 'net income 1' },
-        { value: 'net income 2', text: 'net income 2' },
-    ],
-    occupation_enum: [
-        { value: 'occupation 1', text: 'occupation 1' },
-        { value: 'occupation 2', text: 'occupation 2' },
-    ],
-
-    source_of_wealth_enum: [
-        { value: 'source of wealth 1', text: 'source of wealth 1' },
-        { value: 'source of wealth 2', text: 'source of wealth 2' },
-    ],
-    employment_status_enum: [
-        { value: 'employment status 1', text: 'employment status 1' },
-        { value: 'employment status 2', text: 'employment status 2' },
-    ],
-};
-
 describe('<FinancialDetails />', () => {
-    let mock_props: TFinancialDetails & TFinancialInformationAndTradingExperience = {
+    const mock_props: TFinancialDetails = {
```

Comment:
Let’s not export type for the component, but use it as ComponentProps.
how do you think?